# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../evas/directfb.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- evas/src/bin/evas_directfb_window.c.orig	2000-01-01 00:00:00.000000000 +0100
+++ evas/src/bin/evas_directfb_window.c	2005-10-26 09:28:07.000000000 +0200
@@ -75,7 +75,7 @@
      IDirectFBEventBuffer   *buffer;
 
      DFBDisplayLayerConfig  layer_config;
-     DFBCardCapabilities    caps;
+     DFBGraphicsDeviceDescription cardDescription;
      IDirectFBWindow*       upper;
      DFBWindowID            id1;
 
@@ -88,12 +88,12 @@
      DFBCHECK(DirectFBInit( &argc, &argv ));
      DFBCHECK(DirectFBCreate( &dfb ));
 
-     dfb->GetCardCapabilities( dfb, &caps );
+     dfb->GetDeviceDescription( dfb, &cardDescription );
 
      dfb->GetDisplayLayer( dfb, DLID_PRIMARY, &layer );
 
-     if (!((caps.blitting_flags & DSBLIT_BLEND_ALPHACHANNEL) &&
-           (caps.blitting_flags & DSBLIT_BLEND_COLORALPHA  )))
+     if (!((cardDescription.blitting_flags & DSBLIT_BLEND_ALPHACHANNEL) &&
+           (cardDescription.blitting_flags & DSBLIT_BLEND_COLORALPHA  )))
      {
           layer_config.flags = DLCONF_BUFFERMODE;
           layer_config.buffermode = DLBM_BACKSYSTEM;
--- evas/src/bin/evas_directfb_main.c.orig	2000-01-01 00:00:00.000000000 +0100
+++ evas/src/bin/evas_directfb_main.c	2005-10-26 09:40:39.000000000 +0200
@@ -26,7 +26,6 @@
    IDirectFBEventBuffer *buffer;
    DFBResult           err;
    DFBSurfaceDescription dsc;
-   DFBCardCapabilities caps;
    DFBDisplayLayerConfig layer_config;
    DFBInputEvent       evt;
 
